### PR TITLE
[WIP] Use head -c to shorten the text

### PR DESCRIPTION
**Pull request by mik-laj**

### CODE REVIEW

File: run_environment.sh [hunk 1]
```diff
@@ -572,7 +572,7 @@ AIRFLOW_BREEZE_TEST_SUITE=${ASCII_USER:0:6}${NUMERIC_PYTHON_VERSION}
 RANDOM_FILE=${MY_DIR}/.random
 
 if [[ ! -f ${RANDOM_FILE} ]]; then
-    head /dev/urandom && true | env LC_CTYPE=C tr -dc 'a-z0-9' | fold -w 7 | head -n 1 > ${RANDOM_FILE}
```
Comment (potiuk) [inline run_environment.sh:575]: Why do we need that @mik-laj - I thought -c 7 does the job.

Comment (mik-laj) [reply run_environment.sh:575]: Code readability only. instead of two commands we can use only one. Is the world beautiful now?

**Comment by potiuk** [comment]
I think we should close this one @mik-laj

**Review by potiuk — APPROVED** [review]

File: run_environment.sh (hunk 1 above)
Comment (potiuk) [reply run_environment.sh:575]: It's been already even more simplified in master (using date | md5sum which is much better as it does not require randomness of /dev/urandom)

**Comment by potiuk** [comment]
Closing it - it's already improved in master.

**Pull request closed without merging**